app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ActiveRecord::Base
   has_many :instances, through: :instance_users
   has_many :course_users, inverse_of: :user, dependent: :destroy
   has_many :courses, through: :course_users
+  has_many :group_users, inverse_of: :user, dependent: :destroy, class_name: Course::GroupUser.name
```

Comment:
I think then it also becomes inconsistent? because its class name is <code>Course::GroupUser</code>, which means that it's scoped.

Just like <code>Course::Announcement</code>, we simply use <code> has_many :announcements</code> instead of <code> has_many :course_announcements</code>